environment-wasm-build.yml

```diff
@@ -3,6 +3,4 @@ channels:
   - conda-forge
 dependencies:
   - cmake
-  - emsdk >=3.1.11
```

Comment:
Yes 👍🏽 This one was a nasty hack that I don't want to maintain 😄

Reply:
thanks for the review.

Feel free to merge once you're happy ;)